### PR TITLE
feat: update lib with latest proto

### PR DESCRIPTION
**Pull request by yashrajdesai**
Updated the proto files using these [proto](https://github.com/hyperledger/iroha/tree/main/shared_model/schema) files.
To generate the ts proto files I used the following script:
```
PROTOC_GEN_TS_PATH="../../node_modules/.bin/protoc-gen-ts"

OUT_DIR="./"

protoc \
    --plugin="protoc-gen-ts=${PROTOC_GEN_TS_PATH}" \
    --js_out="import_style=commonjs,binary:${OUT_DIR}" \
    --ts_out="${OUT_DIR}" \
    *.proto
```
Signed-off-by: yashrajdesai <yashrajdesai30@gmail.com>

### CODE REVIEW

**Comment by baziorek** [comment]
@yashrajdesai What is version of `protoc` in your computer?

**Review by baziorek — APPROVED** [review]
Changes look good to me, unfortunately I'm not javascript developer, so somebody needs to take a look from Iroha Core Team.

**Comment by yashrajdesai** [comment]
> @yashrajdesai What is version of `protoc` in your computer?

libprotoc 3.20.1

File: src/proto/endpoint_pb.js
Comment (6r1d) [inline src/proto/endpoint_pb.js:298]: Again, not sure if a full description is necessary. If this is an auto-generated code that'll be discarded, comments are useless. If it's a library code, comments would help.

**Review by 6r1d — APPROVED** [review]
LGTM in general, but if the whole code is not auto-generated, commenting the pieces that could be descriptive would help.

File: src/proto/endpoint_pb.js
Comment (yashrajdesai) [reply src/proto/endpoint_pb.js:298]: The comments were also auto-generated along with the code.

File: src/commands/index.ts
Comment (yashrajdesai) [reply src/commands/index.ts:66]: Yes if it's not significant we can remove the docstring. I added the docstring for 'syncingPeer' thinking it was a convention to document the arguments of the function.

**Pull request merged**
